apps/opik-frontend/src/constants/playground.ts

```diff
@@ -0,0 +1,124 @@
+import OpenAIIcon from "@/icons/integrations/openai.svg?react";
+import { PLAYGROUND_MODEL, PLAYGROUND_PROVIDER } from "@/types/playground";
+
+// @ToDo: remove it
+export const OPENAI_API_KEY = "OPENAI_API_KEY";
+
+export const PLAYGROUND_PROVIDERS = {
+  [PLAYGROUND_PROVIDER.OpenAI]: {
+    title: "Open AI",
+    value: PLAYGROUND_PROVIDER.OpenAI,
+    icon: OpenAIIcon,
+  },
+};
+
+export const PLAYGROUND_MODELS = {
+  [PLAYGROUND_PROVIDER.OpenAI]: [
```

Comment:
Add link above from where we should get this list so in case there are new models we can update the list